usersHandler.js

```diff
@@ -34,22 +37,47 @@ function usersHandler(){
     });
 
     this.generateAPIKey = () => {
-        let key = shortId.generate()
-        this.apps[key] = {
-            calls_per_day : 0,
-            transactions:[],
-        }
-        return key;
-    }
-
-    // TODO: Validate that the API Key does not have too many calls
-    this.validKey = (key) => {
-        return this.apps[key] !== undefined;
+        let newKey = shortId.generate();
+        let client = new MongoClient(uri, {useNewUrlParser : true, useUnifiedTopology : true});
+        client.connect().then(()=>{
```

Comment:
These lines could use some async-await to reduce lines and improve readability.
Everything that uses a then can instead use an await, as long as the function has the async modifier.